Examples/Algorithms/TrackFinding/src/AmbiguityResolutionAlgorithm.cpp

```diff
@@ -191,8 +197,9 @@ ActsExamples::ProcessCode ActsExamples::AmbiguityResolutionAlgorithm::execute(
       tips.push_back(tip);
       parameters.emplace(tip, trackParameters[iTrack]);
     }
-
-    outputTrajectories.emplace_back(traj.multiTrajectory(), tips, parameters);
+    if (!tips.empty()) {
+      outputTrajectories.emplace_back(traj.multiTrajectory(), tips, parameters);
+    }
```

Comment:
I think the idea here was to keep empty trajectories /cc @paulgessinger

Reply:
The reason why I added this is that if we don't and we add the cutoff on the number of hits per track they we have warning in the CI and it fail. I would be happy to change it back though

Reply:
Not sure I get what you're saying.

Reply:
Sorry, without `if (!tips.empty()) {` you try to write empty trajectory if `Nmeas<Nminmeas` for a track, which is true many track. Writing a empty track cause a Warning and the CI is configure to fail in case of warning of the full chain. So, now that I have added a cutoff on the minimum number of hit per track before the solver, we would get Warning in the CI, making it fail.